### DISCUSSION TITLE
absolute imports works fine in VScode but not compiled

### DISCUSSION THREAD

**Discussion by mhesham32** [vercel/next.js#16114] [Help] [answered]
that is my configs in tsconfig.json file 
```json
 "baseUrl": ".",
    "paths": {
      "@src/*": ["./src/*"],
      "@hooks/*": ["./src/hooks/*"],
      "@components/*": ["./src/components/*"],
      "@root/*": ["./*"]
    }
```
these are recognized by vscode and it auto completes the directories for me but I got this error in the browser and terminal 

```
Module not found: Can't resolve '@src/components/AppHeader' in '/mnt/e/dev/projects/share-thinking/pages'
```
however, I'm using the latest next package 9.5

**Comment by jamesmosier** [comment, marked as answer]
On `9.5.2` with my `tsconfig.json` and pages setup like this, it works:

```json
{
    "compilerOptions": {
      "baseUrl": ".",
      "paths": {
        "@/components/*": ["./src/components/*"]
      }
    }
  }

```

`src/pages/index.js`

```jsx
import Test from '@/components/test';

export default function Home() {
  return (
    <div>
      <Test />
    </div>
  );
}
```

`src/components/test.js`

```jsx
export default function Test() {
  return <p>test</p>
}
```



Could you import your AppHeader via `import AppHeader from '@/components/AppHeader';` instead of `import AppHeader from '@/src/components/AppHeader';`?

**Reply by mhesham32** [reply]
thanks for replying 🙏. it is the same thing when I use `import AppHeader from '@/components/AppHeader';` to be clear I set next to latest in package.json when I hover over it It shows the version 9.5 however when I use `yarn next -v ` I got 9.3 what is the right version here could this be the problem if though how to update it.

**Reply by jamesmosier** [reply]
Change your `package.json` to `"next": "9.5.2"` and then run `yarn install` and try again.
